### PR TITLE
Generic Custom Branding

### PR DESCRIPTION
**Pull request by dpose**
Generic Custom Branding

### CODE REVIEW

File: src/plugins/kibana/public/kibana.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const chrome = require('ui/chrome');
 const routes = require('ui/routes');
 const modules = require('ui/modules');
 
-const kibanaLogoUrl = require('ui/images/kibana.svg');
+var kibanaLogoUrl = require('ui/images/bitergia-analytics-logo.png');
```
Comment (sduenas) [inline src/plugins/kibana/public/kibana.js]: Why are you replacing a `const` by a `var`?

File: src/ui/public/styles/variables/bootstrap-mods.less [hunk 2]
```diff
@@ -370,7 +370,7 @@
 @navbar-inverse-border:                 darken(@navbar-inverse-bg, 10%);
 
 // Inverted navbar links
-@navbar-inverse-link-color:             @gray-lighter;
+@navbar-inverse-link-color:             #ED9C06;
```
Comment (sduenas) [inline src/ui/public/styles/variables/bootstrap-mods.less]: Maybe you can define a constant here like they do with `@gray-lighter`. In fact, you have defined the same value for `@brand-info` in `colors.less` file.

File: src/plugins/kibana/public/kibana.js (hunk 1 above)
Comment (acs) [inline src/plugins/kibana/public/kibana.js]: This code comes from Kibana. Probably it was var when we start working on it, then they changed to const. It is better probably to use const.

File: src/ui/public/styles/variables/bootstrap-mods.less (hunk 2 above)
Comment (acs) [inline src/ui/public/styles/variables/bootstrap-mods.less]: Yes, something like @bitergia-orange :)

**Comment by acs** [comment]
In this branch, kibiter with edit support,  we are not going to apply any branding because we want to look it as closer as possible to Kibana upstream. So time to close this PR.

Follow the branding in https://github.com/grimoirelab/kibiter/pull/2

**Pull request closed without merging**